packages/vc-http-api-test-server/package.json

```diff
@@ -3,8 +3,8 @@
   "version": "0.0.1",
   "description": "",
   "scripts": {
-    "test": "dotenv -e .env jest --runInBand --",
-    "run:report": "node runLocal.js",
+    "test": "node runLocal.js",
```

Comment:
this is not going to work since ci is running "npm run run:report" but you removed the command

Reply:
I think the reference should be fixed now

Reply:
Nice, I think you'll need to trigger the cd action to check if it works before we can review

Reply:
Makes sense, I tried running the branch but it hangs on the tests before it gets to publish, might have to do with the bbs issue you mentioned.

Reply:
Yes I think it is related, I have some code on my branch to abort the request after 1second as a temporary fix so that the tests can pass. I will push it if you want to use that to test cd

Reply:
https://github.com/transmute-industries/vc-http-api-1/pull/6/files

Reply:
Yeah that works

Reply:
Says it successfully published but page 404's maybe I am missing something? https://transmute-industries.github.io/vc-http-api-1/test-suite/transmute/index.html